### PR TITLE
2021 07 07 capability readme

### PR DESCRIPTION
**Pull request by rszcodronski**
Adding a ReadME for Capabilities to capture the work completed around modeling and tagging.

Note - I have also added several properties which are described at the end of the Readme regarding "values" metadata.

Is it okay to leave "unit" as a String for now or should this be and enum of all the possible units that exist in the DTDL Semantic Types or defined elsewhere?

### CODE REVIEW

**Review by alinamstanciu — APPROVED** [review]
How would people find the readme.md in capability folder?
We need a phrase in the main doc which links to this readme, otherwise it's hard to find it.
Could you please add that phrase? Thanks

File: Ontology/Capability/README.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+When the digital twin connects to a system or source of data, a capability twin maintains the current value (or latest value received) and a time series record of historical values.
+
+It's important to understand the difference between a `capability` and other entities such as an `asset`. A `capability` is a virtual concept while an `asset` is a physical object. For example, a physical asset of a `sensor equipment` may have one or more capabilities such as `occupancy sensor`, `temperature sensor`, and `illuminance sensor`.
+
+This guide will provide an understanding of how to become familiar with the capabiltiy data model, define capabilty twins and recommendations for how to best use the ontology.
+
+## Capabiltiy Models
```
Comment (hammar) [inline Ontology/Capability/README.md:13]: Capabiltiy -> Capability

File: Ontology/Capability/README.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+When the digital twin connects to a system or source of data, a capability twin maintains the current value (or latest value received) and a time series record of historical values.
+
+It's important to understand the difference between a `capability` and other entities such as an `asset`. A `capability` is a virtual concept while an `asset` is a physical object. For example, a physical asset of a `sensor equipment` may have one or more capabilities such as `occupancy sensor`, `temperature sensor`, and `illuminance sensor`.
+
+This guide will provide an understanding of how to become familiar with the capabiltiy data model, define capabilty twins and recommendations for how to best use the ontology.
```
Comment (hammar) [inline Ontology/Capability/README.md:11]: capabiltiy  -> capability
capabilty -> capability

File: Ontology/Capability/README.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+| ---------- | ----------- |
+| Binary | Boolean values `true` or `false` |
+| Percent | Number values between `0...1` |
+| Multi-State | `String` values often defined by an enumeration |
+
+Each of the above State Kind classes are further sub-classed to give context to the type of State that is being sensed, actuated, configured, or maintained. The State Kind subclass names (i.e. OnOff, Mode, and Position) are common across each of the top-level Capability sub-classes of `Sensor`, `Actuator`, and `Parameter\Setpoint`. However, these top-level sub-classes do not use multiple inheritance of a common StateKind class. As such, clients can query across these top-level classes (i.e. TemperatureSensor and TemperatureSetpoint) by using a string match function such as `STARTSWITH`.
+
+## Capability Properties
+The Capability models maintain several **properties** which allow additional metadata on the twin to provide context to 1) classificaiton 2) values and 3) communication configuration. The properties also maintain the latest value the twin received from the connected entity (`lastValue`) and the time at which it was sampled or reported by the connected entity (`lastValueTime`).
```
Comment (hammar) [inline Ontology/Capability/README.md:58]: classificaiton -> classification

File: Ontology/Capability/README.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+### Tags Properties
+The Capabiltiy models described above provide a great means of classifying based on **function** and **kind** which brings consistency and inheritence to these two common means by which a Capability is defined and would be queried. However, most impelementations of a digital twin desire additional metadata context to query, analyze, and filter amongs twins of the same model. Because there are many ways in which a capability could be classified in the real world, there are too many combinations to pre-define models for every possible real-world scneario. The use of properties within the **tags** component allows the flexibility to add context to the base set of models which have been defined above.
```
Comment (hammar) [inline Ontology/Capability/README.md:61]: Capabiltiy -> Capability

**Review by hammar — CHANGES_REQUESTED** [review]
Great text, good work Rick! I added a couple of minor spelling issues that you may want to fix, but overall, this looks good to me. Very useful for implementors, I'm sure.

1. I'd suggest placing this file outside of the Ontology directory; to avoid the risk of it being overwritten by automated tools (e.g., the translation mechanism).
2. I'd also suggest not including the updated properties in the same PR, since we've already reviewed the other PR for those, and I think a little more work is needed on them.

**Pull request closed without merging**